dask_ml/model_selection/_incremental.py

```diff
@@ -218,13 +220,20 @@ def get_futures(partial_fit_calls):
         This function handles that policy internally, and also controls random
         access to training data.
         """
+        if dask.is_dask_collection(y_train):
```

Comment:
I think this should only check `X_train` because `y_train` is an optional argument.

Reply:
I've done some tracing and it appears that passing `y=None` into `search.fit` isn't supported? That's unrelated to this PR.